docs/howto/pipeline_testing.md

```diff
@@ -99,6 +101,8 @@ numeric_keyword_fields:
 
 The `multiline` section ([raw files](#raw-files) only) configures the log file reader to correctly detect multiline log entries using the `first_line_pattern`. Use this property if your logs may be split into multiple lines, e.g. Java stack traces.
 
+The `exclude_lines` sections allows to filter out lines from the sample log files. If `multiline` is used, pattern is checked after merging lines.
```

Comment:
Regarding the https://github.com/elastic/integrations/pull/1145/: 

I'm wondering if it's possible to remove lines starting with `#` with ingest pipeline (during processing, just skip them).

Reply:
Good idea, I will try that!

Reply:
It seems that something like https://github.com/elastic/elastic-package/pull/384 will be needed, otherwise errors like this happen:
```
ERROR: verifying test result failed: comparing test results failed: can't adjust test results: can't unmarshal event: unexpected end of JSON input
```